src/pypdfium2/_helpers/page.py

```diff
@@ -262,7 +269,7 @@ def get_objects(self, filter=None, max_depth=2, form=None, level=0):
             :class:`.PdfObject`: A page object.
         """
```

Comment:
note to folded code about: We may want to bump the default of max_depth from 2 to 15 to align with the extract-images CLI